apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -77,9 +77,6 @@ export const submitChatMessage = createAsyncThunk(
     const systemPrompt = generalChatSystemPrompt;
     const storedMessages = state.aiTutor.chatMessages;
     const newMessageId = storedMessages[storedMessages.length - 1].id + 1;
-    const appropriateChatMessages = storedMessages.filter(
-      msg => msg.status === Status.OK
-    );
```

Comment:
We show all the messages now, just hide content for inappropriate messages and style them differently.